pom.xml

```diff
@@ -990,9 +990,9 @@
           <version>1.5.1</version>
           <configuration>
             <printEqualFiles>false</printEqualFiles>
-            <failBuildInCaseOfDifferentContentConflict>false</failBuildInCaseOfDifferentContentConflict>
-            <failBuildInCaseOfEqualContentConflict>false</failBuildInCaseOfEqualContentConflict>
-            <failBuildInCaseOfConflict>false</failBuildInCaseOfConflict>
+            <failBuildInCaseOfDifferentContentConflict>true</failBuildInCaseOfDifferentContentConflict>
+            <failBuildInCaseOfEqualContentConflict>true</failBuildInCaseOfEqualContentConflict>
```

Comment:
Are we sure we should fail the build even for identical content?

Reply:
Nope...probably too strict